src/ResourceManager/SignalR/Commands.SignalR/help/Get-AzureRmSignalR.md

```diff
@@ -38,42 +38,43 @@ Get a specific SignalR service or all the SignalR services in a resource group o
 ```powershell
 PS C:\> Get-AzureRmSignalR
 
-HostName                                           Location       ServerPort PublicPort ProvisioningState
---------                                           --------       ---------- ---------- -----------------
-mysignalr1.servicedev.signalr.net                  eastus         5002       5001       Succeeded
-mysignalr2.servicedev.signalr.net                  eastus         5002       5001       Succeeded
-mysignalr3.servicedev.signalr.net                  eastus         5002       5001       Creating
+
+HostName                                           Location       ServerPort PublicPort ProvisioningState Version
+--------                                           --------       ---------- ---------- ----------------- -------
+mysignalr1.service.signalr.net                     eastus         5002       5001       Succeeded         1.0-preview
```

Comment:
@LGDoor I removed the `preview` suffix from the module version (1.0.0-preview --> 1.0.0), so I'm not sure if that had any impact on the version returned -- just want to double-check